packages/backend-core/src/api/resources/JSON.ts

```diff
@@ -133,6 +133,7 @@ export interface InvitationJSON extends ClerkResourceJSON {
 export interface OrganizationJSON extends ClerkResourceJSON {
   object: ObjectType.Organization;
   name: string;
+  slug: string;
```

Comment:
❓ does this always exist, even if not provided by the user during the initial `.create` call?

Reply:
Yes it does. As things currently stand, we'll assign a slug based on the organization name.

Reply:
Requirements have changed and slugs can now be null. Updated the code accordingly. 

See https://github.com/clerkinc/javascript/pull/192/files#diff-2f13a8c2ca4bf45dd6b30b61d8794dee5f585ae33519ffaf846d15380049bc70R136 and https://github.com/clerkinc/javascript/pull/192/files#diff-a04276d17d7525335dfceb4141e67c8ba0fc9b9ad839c185167f9921f2dfd791R78.